<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1154</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1141</width>
      <height>701</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>OpenCV </string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>10</y>
        <width>421</width>
        <height>341</height>
       </rect>
      </property>
      <property name="text">
       <string>After Recognition</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>421</width>
        <height>341</height>
       </rect>
      </property>
      <property name="text">
       <string>Input Image</string>
      </property>
     </widget>
     <widget class="QLabel" name="result_label">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>380</y>
        <width>361</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>The license plate is: </string>
      </property>
     </widget>
     <widget class="QPushButton" name="recognitionButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>390</y>
        <width>321</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Recognize the license plate(s) in the image</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Haar Cascade</string>
     </attribute>
     <widget class="QPushButton" name="recognitionButton_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>390</y>
        <width>321</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Recognize the license plate(s) in the image</string>
      </property>
     </widget>
     <widget class="QLabel" name="result_label_2">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>380</y>
        <width>361</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>The license plate is: </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>421</width>
        <height>341</height>
       </rect>
      </property>
      <property name="text">
       <string>Input Image</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>10</y>
        <width>421</width>
        <height>341</height>
       </rect>
      </property>
      <property name="text">
       <string>After Recognition</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1154</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSelect_file">
    <property name="title">
     <string>Select file</string>
    </property>
    <addaction name="actionChoose_an_image"/>
    <addaction name="actionChoose_an_image_Tab_2"/>
   </widget>
   <addaction name="menuSelect_file"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionChoose_an_image">
   <property name="text">
    <string>Choose an image (Tab 1)</string>
   </property>
  </action>
  <action name="actionChoose_an_image_Tab_2">
   <property name="text">
    <string>Choose an image (Tab 2)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
